bpf/bpf_host.c

```diff
@@ -335,11 +337,20 @@ handle_ipv6_cont(struct __ctx_buff *ctx, __u32 secctx, const bool from_host,
 	dst = (union v6addr *) &ip6->daddr;
 	info = ipcache_lookup6(&IPCACHE_MAP, dst, V6_CACHE_KEY_LEN, 0);
 
+	encrypt_key = 0;
+#if defined(ENABLE_IPSEC)
+	//from_proxy = (ctx->mark & MARK_MAGIC_HOST_MASK) == MARK_MAGIC_PROXY_INGRESS;
+	from_proxy = true;
```

Comment:
We should encrypt from_host packets only if they are from proxy and marked with `0xa00`. But at this time, both skb->mark and skb->cb has been cleared, I couldn't find a way to check mark.

Possible solutions:
a. find another position where we can get mark (`magic` variable). 
b. find a way to pass mark or some variable to this position.

Options B seems to be easier, because at this position we can leverage existing functions and `info` from ipcache.

@pchaigno Do you have any idea?